VERSION.md

```diff
@@ -1 +1 @@
-0.0.12
+0.0.13
```

Comment:
Please also make sure this is included in the updates after you pull/rebase against `origin/main`